.kokoro/system_tests.sh

```diff
@@ -2,6 +2,7 @@
 
 source $KOKORO_GFILE_DIR/secrets.sh
 
+# Get a project from the project pool.
```

Comment:
Is this tool documented externally? Or internally at least?

Reply:
Added a link to the implementation.